Documentation/common-problems.md

```diff
@@ -4,22 +4,66 @@ weight: 78
 
 If after trying the suggestions found on this page and the problem is not resolved, the Rook team is very happy to help you troubleshoot the issues in their Slack channel. Once you have [registered for the Rook Slack](https://rook-slackin.herokuapp.com/), proceed to the General channel to ask for assistance.
 
 ## Table of Contents
+- [Troubleshooting Techniques](#troubleshooting-techniques)
 - [Pod using Rook storage is not running](#pod-using-rook-storage-is-not-running)
 - [Cluster failing to service requests](#cluster-failing-to-service-requests)
 - [Only a single monitor pod starts](#only-a-single-monitor-pod-starts)
 - [OSD pods are failing to start](#osd-pods-are-failing-to-start)
 - [Node hangs after reboot](#node-hangs-after-reboot)
 
 # Troubleshooting Techniques
-One of the first things that should be done is to start the [rook-tools pod](./toolbox.md) as described in the Toolbox section. Once the pod is up and running one can `kubectl exec` into the pod to execute Ceph commands to evaluate that current state of the cluster. Here is a list of commands that can help one get an understanding of the current state.
+There are two main categories of information you will need to investigate issues in the cluster:
+1. Kubernetes status and logs
+1. Ceph status
+
+## Kubernetes Tools
+Kubernetes status will be the first line of investigating when something goes wrong with the cluster. Here are a few artifacts that would be helpful to gather:
+- Rook pod status: 
+  - `kubectl get pod -n rook -o wide`
+  - `kubectl get pod -n rook-system -o wide`
+- Logs for Rook pods
+  - Logs for the operator: `kubectl logs -n rook-system -l app=rook-operator`
+  - Logs for a specific pod: `kubectl logs -n rook <pod-name>`, or a pod using a label such as mon1: `kubectl logs -n rook -l mon=rook-ceph-mon1`
+  - Logs on a specific node to find why a PVC is failing to mount:
+    - Rook agent errors around the attach/detach: `kubectl logs -n rook-system <rook-agent-pod>`
+    - Connect to the node, then get kubelet logs (if your distro is using systemd): `journalctl -u kubelet`
+- Other Rook artifacts:
+  - The monitors that are expected to be in quorum: `kubectl -n rook get configmap rook-ceph-mon-endpoints -o yaml | grep data`
+  - More artifacts in the `rook` namespace: `kubectl -n rook get all`
+  
+## Ceph Tools
+After you verify the basic health of the running pods, next you will want to run Ceph tools for status of the storage components. There are two ways to run the Ceph tools, either in the Rook toolbox or inside other Rook pods that are already running.
+
+### Tools in the Rook Toolbox
+ The [rook-tools pod](./toolbox.md) is a one-stop shop for both Ceph tools and other troubleshooting tools. Once the pod is up and running one can `kubectl exec` into the pod to execute Ceph commands to evaluate that current state of the cluster. 
```

Comment:
may as well put the full command inline here for simplicity: `kubectl -n rook exec -it rook-tools bash`

Reply:
Good idea, i'll expand it